torchvision/utils.py

```diff
@@ -187,6 +188,7 @@ def draw_bounding_boxes(
         font_size (int): The requested font size in points.
         label_colors (color or list of colors, optional): Colors for the label text.  See the description of the
             `colors` argument for details.  Defaults to the same colors used for the boxes.
+        fill_labels (bool): If `True` fills the label background with specified color. Default: False.
```

Comment:
Minor suggestion, as I originally was wondering whether this was `colors` or `label_colors` (which is just for  the text)

```suggestion
        fill_labels (bool): If `True` fills the label background with specified box color (from the ``colors`` parameter). Default: False.
```